snekbox/__main__.py

```diff
@@ -37,7 +37,7 @@ def parse_args() -> argparse.Namespace:
 def main() -> None:
     """Evaluate Python code through NsJail."""
     args = parse_args()
-    result = NsJail().python3(args.code, nsjail_args=args.nsjail_args, py_args=args.py_args)
+    result = NsJail().python3(py_args=[*args.py_args, args.code], nsjail_args=args.nsjail_args)
```

Comment:
`py_args` needs to default to `-c` here to preserve the original behaviour, since `python3()` no longer has it as a default. (I think this is the most convenient default when using it via the CLI for testing).

Reply:
I think the argparse default here already makes `-c` the default? The previous unit tests for main also work fine.